### ISSUE TITLE
534ez - Staging Review finding: Focus remains on Continue and Back buttons

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#124516]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Bio-Huntridgelabs
**Product:** VA Form 21P-534EZ
**Feature:** Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** Yes
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

In Safari, with Voiceover, programmatic focus is placed correctly on the Progress Bar - Segmented, however Voiceover cursor remains on the Continue or Back buttons, so Voiceover users aren't given an indication that they are on a new page, which may be confusing. 

Note: Safari cursor is handled correctly when the step is a single entry step - e.g. "Do you have a dependent child to add?" - Focus and Voiceover cursor/focus are both on the heading. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="1504" height="1478" alt="SPA focus issue" src="https://github.com/user-attachments/assets/ad39c29f-5a6d-4ee1-8064-28e58a1cceb7" />

</details>

## Recommended action

Ensure focus is managed appropriately as navigation between screens occurs. 

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by maggiewachs** [comment]
Verifying whether this is a WCAG failure or fixable. Will update asap.

**Comment by maggiewachs** [comment]
This issue impacts design system components and will be tracked here: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/5576

**Comment by caw310** [comment]
@maggiewachs [[Accessibility]: VoiceOver cursor does not match programmatic focus in Safari](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/5576) is being worked on in our current sprint which started today 2/16.
